scripts/replace-all-mocks.py

```
@@ -0,0 +1,169 @@
+    """Add necessary imports to test files"""
+    if "test_" in file_path.name:
+        content = file_path.read_text()
+
+        # Check if import already exists
+        if "from config.mcp_config import MCPConfig" not in content:
+            # Add import after other imports
+            lines = content.split("\n")
+            import_added = False
+
+            for i, line in enumerate(lines):
+                if line.startswith("import ") or line.startswith("from "):
```

Ruff (0.11.9)

Call `startswith` once with a `tuple`

Merge into a single `startswith` call

(PIE810)